staging/src/k8s.io/api/admissionregistration/v1beta1/types.go

```diff
@@ -265,6 +265,11 @@ type WebhookClientConfig struct {
 	// the webhook's server certificate.
 	// Required.
 	CABundle []byte `json:"caBundle" protobuf:"bytes,2,opt,name=caBundle"`
+
+	// `timeoutSeconds` is the number of seconds after which the webhook request times out.
+	// Defaults to 1 second. Minimum value is 1.
+	// +optional
+	TimeoutSeconds int32 `json:"timeoutSeconds,omitempty" protobuf:"varint,4,opt,name=timeoutSeconds"`
```

Comment:
this should be `metav1.Duration`. Should it default to 1 or shouldn't it be an optional value (a pointer) and then default dynamically to the server default?

Reply:
referenced from `Probe`